src/cksum/cksum.c

```diff
@@ -0,0 +1,101 @@
+#include<stdint.h>
```

Comment:
Can you run `clang-format`? This enforce coding style consistency across all sources